src/elements/content-sidebar/versions/VersionsList.scss

```diff
@@ -11,6 +11,17 @@
     }
 }
 
+.bcs-VersionsList-heading {
+    color: $bdl-gray-50;
+    font-size: 14px;
+    padding-bottom: 10px;
+    padding-top: 15px;
+
+    &:empty {
+        display: none;
```

Comment:
When would this occur?

Reply:
It won't; it's a stray declaration from an earlier revision. Good catch, will remove.